client/accounting.go

```diff
@@ -44,13 +44,18 @@ func (x ResBalanceGet) Amount() accounting.Decimal {
 // Context is required and must not be nil. It is used for network communication.
 //
 // Return errors:
-//   - [ErrMissingAccount]
+//   - [ErrMissingAccount];
+//   - [ErrMissingSigner].
```

Comment:
Do we need punctuation in these lists?

Reply:
Not sure, but I'm already seen such punctuation in some another func docs. And left it as was, when updated docs. We just should decide - use it or not. My opinion it is redundantly. Moreover if we decide to remove it, we may do this in #409 or in any PR, where we are touching the documentaion

Reply:
It seems to be a little excessive for me, we can't properly typeset this doc, so I doubt it's worth trying. I couldn't find any best practices for this either. @notimetoname, any reason _why_ you want it this way?

Reply:
> any reason why you want it this way?

just the first example i saw in that repo _had_ that chars. if my first experience was without them, you would see another version. will drop them

Reply:
done